src/org/jitsi/util/ArrayUtils.java

```diff
@@ -63,6 +63,25 @@
         return array;
     }
 
+    public static <T> T[] insert(T element, T[] array, Class<T> componentType)
+    {
+        boolean inserted = false;
+        for (int i = 0; i < array.length; ++i)
+        {
+            if (array[i] == null)
+            {
+                array[i] = element;
+                inserted = true;
```

Comment:
Looks like this inserts the element in ALL null fields of the array. Also, docs.

Reply:
right you are...fixed